### PR TITLE
Update to OTel SDK v0.10.0

### LINKED ISSUE #77: otel v0.10.0 support
I am having issues with otel version.
can you please update to go.opentelemetry.io/otel v0.10.0

I tried with go.mod replace
```
replace github.com/GoogleCloudPlatform/opentelemetry-operations-go/exporter/trace v0.2.1 => github.com/gzhao408/opentelemetry-operations-go/exporter/trace v0.2.2-0.20200727182237-86360c493900

replace github.com/GoogleCloudPlatform/opentelemetry-operations-go/exporter/metric v0.2.1 => github.com/gzhao408/opentelemetry-operations-go/exporter/metric v0.2.2-0.20200727182237-86360c493900
```
but getting error

```
go: finding module for package go.opentelemetry.io/otel/api/kv/value
../../../../go/pkg/mod/github.com/gzhao408/opentelemetry-operations-go/exporter/trace@v0.2.2-0.20200727182237-86360c493900/trace_proto.go:31:2: module go.opentelemetry.io/otel@latest found (v0.10.0), but does not contain package go.opentelemetry.io/otel/api/kv/value
```

### PR DESCRIPTION
**Pull request by nilebox**
Fixes #77

- Added `make update-otel` target to make it easier to update all Go modules at once
- Updated to v0.10.0
- Fixed import of moved packages

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -109,3 +109,28 @@ lint: $(TOOLS_DIR)/golangci-lint $(TOOLS_DIR)/misspell
...
+
+.PHONY: update-dep
+update-dep:
+	$(MAKE) for-all CMD="$(PWD)/internal/buildscripts/update-dep"
+	$(MAKE) build
+	$(MAKE) gotidy
+
+OTEL_VERSION=v0.10.0
+
+.PHONY: update-otel
```
Comment (nilebox) [inline Makefile:134]: Based on https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/5e2a04c395b270b6767d757b4697f56632458062/Makefile#L155-L157

File: Makefile [hunk 2]
```diff
@@ -109,3 +109,28 @@ lint: $(TOOLS_DIR)/golangci-lint $(TOOLS_DIR)/misspell
...
+gotidy:
+	$(MAKE) for-all CMD="go mod tidy"
+
+.PHONY: update-dep
+update-dep:
+	$(MAKE) for-all CMD="$(PWD)/internal/buildscripts/update-dep"
+	$(MAKE) build
+	$(MAKE) gotidy
+
+OTEL_VERSION=v0.10.0
```
Comment (nilebox) [inline Makefile:132]: Pinned version since we want to use released versions

File: example/metric/go.mod [hunk 3]
```diff
@@ -6,6 +6,7 @@ replace github.com/GoogleCloudPlatform/opentelemetry-operations-go/exporter/metr
 
 require (
 	github.com/GoogleCloudPlatform/opentelemetry-operations-go/exporter/metric v0.2.1
-	go.opentelemetry.io/otel v0.9.0
+	go.opentelemetry.io/otel v0.10.0
+	go.opentelemetry.io/otel/sdk v0.10.0
```
Comment (james-bebbington) [inline example/metric/go.mod:10]: How come this got added? Seems weird given that no code changed (likewise for elsewhere)

File: .gitignore [hunk 4]
```diff
@@ -9,3 +9,4 @@ Thumbs.db
 *.so
 coverage.*
 
+/example/metric/metric
```
Comment (james-bebbington) [inline .gitignore:12]: What is that file btw?

File: example/metric/go.mod (hunk 3 above)
Comment (james-bebbington) [reply example/metric/go.mod:10]: Nvm just saw https://github.com/open-telemetry/opentelemetry-go/pull/985

File: .gitignore (hunk 4 above)
Comment (nilebox) [reply .gitignore:12]: It's a binary generated during the build I think. `make precommit` complained about this file appearing and being uncommitted.

**Pull request merged**